samples/WindowsScript/WindowsScript/WindowsScript.ProjectType/BuildSystem/Rules/ProjectItemsSchema.xaml

```diff
@@ -33,6 +39,7 @@
         ItemType="Compile">
     </ContentType>
 
+    <ItemType Name="Script" DisplayName="Script"/>
     <ItemType Name="Compile" DisplayName="C# compiler"/>
```

Comment:
Clean up this file to remove the unused content, items, and extensions. (Compile for example).